module/VuFind/src/VuFind/Db/Table/Resource.php

```diff
@@ -254,7 +252,6 @@ public function updateRecordId($oldId, $newId, $source = DEFAULT_SEARCH_BACKEND)
                         ['resource_id' => $resource->id]
                     );
                 }
-                $resource->delete();
```

Comment:
Why are you removing this line? I think we do want to delete the old resource after associated rows have been migrated to the new resource.

Reply:
Sorry, that's a mistake, I just found it, working out of doctrine branch will fix this in PR on deduplication.